index.js

```diff
@@ -1,17 +1,28 @@
 "use strict";
 
 function assignSender(context) {
-  context.github.issues.addAssigneesToIssue(context.issue({
-    assignees: [context.payload.sender.login]
-  }));
+  context.github.issues.addAssigneesToIssue(
+    context.issue({
+      assignees: [context.payload.sender.login],
+    })
+  );
 }
 
 module.exports = (robot) => {
+  robot.on(
+    ["issues.opened", "issues.edited", "issues.unassigned"],
+    async (context) => {
+      const config = await context.config("config.yml", {

```

Comment:
I think I'd cal the file `always-assign.yml` so we have separate files per plugin. I think that could be more future proof than a general `config.yml`.
```suggestion
      const config = await context.config("always-assign.yml", {

```